src/kOS/Module/kOSVesselModule.cs

```diff
@@ -67,7 +68,7 @@ public override void OnAwake()
         /// KSP after Awake.  All parts should be added to the vessel now, so it is safe to walk the
         /// parts tree to find the attached kOSProcessor modules.
         /// </summary>
-        public void Start()
+        protected override void OnStart()
```

Comment:
It might be an effect of the fact that Unity views C# as an "also" supported language rather than having been written explicitly for it and it alone.  Using its inheritence rules for base functionality in the API might have made it hard to implement things for other languages.